packages/zoe/src/contractFacet/vatRoot.js

```diff
@@ -8,51 +8,84 @@
 // `yarn build-zcfBundle`.
 
 import { Far } from '@endo/marshal';
+import { E } from '@endo/far';
 
 import '../../exported.js';
 import '../internal-types.js';
 
 import { makeZCFZygote } from './zcfZygote.js';
 
+const { details: X } = assert;
+
 /**
  * @param {VatPowers & { testJigSetter: TestJigSetter }} powers
- * @returns {{ executeContract: ExecuteContract}}
+ * @param {{contractBundleCap: BundleCap, zoeService: ZoeService, invitationIssuer: Issuer, privateArgs: any}} vatParameters
```

Comment:
Are we allowed to omit `privateArgs`? If so we should declare them optional.

```suggestion
 * @param {{contractBundleCap: BundleCap, zoeService: ZoeService, invitationIssuer: Issuer, privateArgs?: any}} vatParameters
```

Reply:
done